docs/_includes/intrinsic_attributes.adoc

```diff
@@ -0,0 +1,467 @@
+
+// Used from annex B Attributes
+
+[[table.intrinsicattributes]]
+
+== Intrinsic attributes
+
+These attributes generally provide instructions to the processor or backend to modify the output.
+They are designed to be set in the header or from the command line.
+They have no effect if they are set in the document body.
+
+.Intrinsic attributes
+[[table.headerattributes]]
+[cols="10,25,10,10,10",width="80%"]
+|====
+|Attribute name |Description |Default value |Acceptable values |More information
+
+|appendix-caption
+|Sets the text used to prefix appendix titles.
+|Appendix
+|Any
+|<<user-appendix>>
+
+|attribute-missing
+|Controls how missing references are handled.
+|skip
+|skip, drop, drop-line, warn
+|<<catch-a-missing-or-undefined-attribute>>
+
+|attribute-undefined
+|Controls how expressions that undefine an attribute are handled.
+|drop-line
+|drop, drop-line
+|<<catch-a-missing-or-undefined-attribute>>
+
+|caution-caption
+|Sets the text used to label CAUTION admonitions, if icons are not used.
+|Caution
+|Any
+|<<admonition>>
+
+// decided to remove this since it is not implemented and may not be an attribute even if it ever is.
+//|compact
+//|Hint to the Docbook backend to produce close spaced lists. NOT YET IMPLEMENTED.
+//|(not set)
+//|Any (value not used)
+//|-
+
+|copycss
+|Copy and link, rather than embed, the CSS files for HTML backends.
+|(empty)
+|Any (value not used)
+|?
+
+|customcss
+|Replace the default stylesheet for HTML backends.
+|(not set)
+|Path
+|<<deckjs-backend-attributes>>
+
+|customjs
+|
+|(not set)
+|
+|<<deckjs-backend-attributes>>
+
+|data-uri
+|Embed graphics as data-uri elements in html outputs so that the file is completely self-contained.
+|(not set - graphics are linked, not embedded)
+|
+|<<managing-images>>
+
+|deckjs_theme
+|
+|(not set)
+|
+|<<deckjs-backend-attributes>>
+
+|deckjs_transition
+|
+|(not set)
+|
+|<<deckjs-backend-attributes>>
+
+|docinfo
+|Read input from a Docbook info file.
+|(not set)
+|Any (value not used)
+|<<docinfo-attributes-and-file-names>>
+
+|docinfo1
+|Read input from a Docbook info file.
+|(not set)
+|Any (value not used)
+|<<docinfo-attributes-and-file-names>>
+
+|docinfo2
+|Read input from a Docbook info file.
+|(not set)
+|Any (value not used)
+|<<docinfo-attributes-and-file-names>>
+
+|doctype
+|Set the output document type.
+|article
+|article, book, inline, manpage
+|<<document-types>>
+
+|embedded
+|Is this real? related to inline doctype?
+|(not set)
+|?
+|?
+
+|encoding
+|The encoding of the input and output files, all backends.
+|UTF-8
+|Any valid encoding string
+|-
```

Comment:
encoding is no longer set by default. It can be defined by the user to control the encoding used in the output document. The value of UTF-8 will be used if the attribute is not set, but `{encoding}` will not resolve by default.